source/dem/dem.cc

```diff
@@ -117,6 +124,20 @@ DEMSolver<dim>::print_initial_info()
     << "***************************************************************** \n";
   pcout << "Starting simulation with Lethe/DEM on " << n_mpi_processes
         << " processors" << std::endl;
+  if (simulation_control->get_time_step() > 0.15 * rayleigh_time_step)
+    {
+      pcout << "Warning: DEM time-step is "
+            << (simulation_control->get_time_step() / rayleigh_time_step) * 100
+            << "% of Rayleigh time step" << std::endl;
+      pcout << "It is recommended to decrease the time-step" << std::endl;
+    }
+  else if (simulation_control->get_time_step() < 0.01 * rayleigh_time_step)
+    {
+      pcout << "Warning: DEM time-step is "
+            << (simulation_control->get_time_step() / rayleigh_time_step) * 100
+            << "% of Rayleigh time step" << std::endl;
+      pcout << "It is recommended to increase the time-step" << std::endl;
+    }
```

Comment:
It it a  good idea to suggest increasing or decreasing the time step. However, I would print the value of the fraction no-matter what.
You could print the % of the Rayleigh time step. Then if the value is >0.15 or <0.01 then you could print the warning. It would be clearer and this way the user would always be aware of the Rayleigh time step !